Lib/shlex.py

```diff
@@ -7,11 +7,7 @@
 # iterator interface by Gustavo Niemeyer, April 2003.
 # changes to tokenize more like Posix shells by Vinay Sajip, July 2016.
 
-import os
-import re
 import sys
-from collections import deque
```

Comment:
Is this import time very slow? if not, we can keep it. One reason is that `collections` is usually already present at start-up time (I think?)

Reply:
Only `_collections_abc` is imported at startup (because it's used by `io`). `collections` is somewhat slow, I was just looking to see if I could defer it in typing.py.